### ISSUE TITLE
[Need help] How to install extensions in php egg?

### ISSUE THREAD

**Issue by Miniontoby** [YajTPG/pterodactyl-eggs#27]
Hi,

I just started using this egg, but I need some extensions to be installed/enabled.

I have tried the command `docker-php-ext-install` but it gave me an error: `mkdir: cannot create directory '/usr/src/php': Read-only file system`

Does anyone have any ideas on how to do this?

Kind regards,
- Miniontoby

**Comment by Kuroo2021** [comment]
Give rights to it?

**Comment by LegalSmeagol** [comment]
Yeah I've found no way of achieving this and it feels really silly that the author of this egg made no way to provision for the installation of PHP extensions. Seems half-baked, rushed and amateurish. Try just using Caddy instead of this egg
